cli/tests/pcluster/templates/test_cluster_stack/test_cluster_config_limits/slurm.full_config.snapshot.yaml

```diff
@@ -95,6 +95,15 @@ LoginNodes:
     Image:
       CustomAmi: ami-12345678
     InstanceType: t2.micro
+    LocalStorage:
+      EphemeralVolume:
+        MountDir: /test
+      RootVolume:
+        Encrypted: true
+        Iops: 3000
+        Size: 40
+        Throughput: 125
+        VolumeType: gp3
```

Comment:
This is a big change in the public interface of the product. 
I'm not sure we want to have this public ..at least not without the approval of the PM

Reply:
Done! Thank you for your suggestion, Nicola!